app.js

```diff
@@ -3,25 +3,30 @@ const app = express();
 
 const { getEndpoints } = require("./db/controllers/api.controller.js");
 const { getTopics } = require("./db/controllers/topics.controller.js");
-
 const {
   getArticleById,
-} = require("./db/controllers/articlesbyid.controller.js");
+  getAllArticles,
+} = require("./db/controllers/articles.controller.js");
 
 app.get("/api", getEndpoints);
 
 app.get("/api/topics", getTopics);
 
 app.get("/api/articles/:article_id", getArticleById);
 
+app.get("/api/articles", getAllArticles);
+
 app.use("/*", (req, res) => {
   res.status(404).send({ msg: "Endpoint not found" });
 });
 
 app.use((err, req, res, next) => {
-  const { status = 500, msg = "Internal Server Error" } = err;
-
-  res.status(status).send({ msg });
+  if (err.status && err.msg) {
```

Comment:
Refer back to the error handling notes to set up this middleware more efficiently and follow good separation of concerns: https://l2c.northcoders.com/courses/sd-notes/back-end#sectionId=error-handling,step=intro